content/release-notes/2022-10-07.md

```diff
@@ -0,0 +1,13 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+- Pageserver: Increase default compaction_period setting to 20 s
+- Pageserver: Initial support for online tenant rebalancing
+- Pageserver: Support for multiple postgres versions (14 and 15 for now)
```

Comment:
```suggestion
- Pageserver: Added support for multiple PostgreSQL versions.
```

Reply:
Do we want to announce support for PG 15 here?

Reply:
+1, let's keep it the way you've phrased it